packages/ingest/S3ProviderClient.js

```diff
@@ -7,7 +7,7 @@ const isString = require('lodash/isString');
 const { basename, dirname } = require('path');
 
 class S3ProviderClient {
-  constructor({ bucket }) {
+  constructor({ bucket } = {}) {
```

Comment:
This change allows `new S3ProviderClient()` (as opposed to `new S3ProviderClient({})`) to throw the error instead of complaining about not being able to destructure undefined.